projects/ad-acevsecrdset-sl/src/state_machine/state_machine.c

```diff
@@ -234,6 +245,7 @@ int state_machine()
 			pilot_pwm_timer_set_duty_cycle(stout, PWM_OFF);
 			goto error;
 		}
+		stout->err_status = ret;
```

Comment:
is this a related change to what it is described in the commit body/title?

Reply:
Yes, this change is related to the change in the commit message.